### ISSUE TITLE
[BUG] Pre-compiled app depends on a specific glibc version, so it basically only works on debian buster

### ISSUE THREAD

**Issue by fake-name** [AnalogJ/scrutiny#82]
Going through the [MANUAL_INSTALL.md](https://github.com/AnalogJ/scrutiny/blob/master/docs/INSTALL_MANUAL.md) file, it states:

> Since the webapp is packaged as a stand alone binary, there isn't really any software you need to install other than glibc which is included by most linux OS's already.

This isn't true. I tried to set up the application on ubuntu server 18.04, and it will not run, as the binary depends on glibc 2.28, and ubuntu server 18.04 ships glibc 2.27. Updating to 20.04 gives you glibc 2.30.

Digging around, it looks like the the distro that ships glibc 2.28 is debian buster, so I'm guessing this is the platform you built the software on.

Basically, it could probably be made to work, but would require recompiling the application.

The documentation should probably reflect the platform limitations.

Strangely, the collector doesn't seem to suffer from these issues. Not sure why.

**Comment by fake-name** [comment]
As an aside, here's a systemd unit file for running the web interface (assuming the [readme directory structure](https://github.com/AnalogJ/scrutiny/blob/master/docs/INSTALL_MANUAL.md#directory-structure) and a user/group named `scrutiny` to run under who owns said directories:

```
# File: scrutiny-web.service
[Unit]
Description=Scrutiny web server
Wants=network-online.target
After=network-online.target

[Service]
ExecStart=/opt/scrutiny/bin/scrutiny-web-linux-amd64 start --config /opt/scrutiny/config/scrutiny.yaml
Restart=on-failure
User=scrutiny

[Install]
WantedBy=multi-user.target

```

**Comment by francescayeye** [comment]
@fake-name dependecy to glibc is probably due to usage of net package in the code, even if I find it strange that it's pinned to specific minor version.

there are a few ways to build a static binary without glibc dependency either instructing the build to use go implementation of net package or disabling cgo at all

**Comment by AnalogJ** [comment]
Hey @fake-name @aspacca 
Apologies, you're correct. I'm working on a true static binary right now.

**Comment by fake-name** [comment]
It wasn't a big deal, more of a "maybe this should be in the docs" thing.

Overall, the project is super handy and once the multi-system stuff lands will totally be something I monitor regularly.

**Comment by AnalogJ** [comment]
Hey, do you mind downloading the binaries.zip file from [this CI build](https://github.com/AnalogJ/scrutiny/actions/runs/294936327) and checking if it works for you? Both the webapp and the collector should be fully static now
